### ISSUE TITLE
[jss-compose][jss-nested]

### ISSUE THREAD

**Issue by andrewfan** [cssinjs/jss#918]
https://cssinjs.org/jss-compose?v=v5.0.0#caveats
Are there any plans to marry compose with nested?

**Comment by HenriBeck** [comment]
Why would you need compose inside a nested rule?

**Comment by andrewfan** [comment]
@HenriBeck it is very usefull when you utility first css frameworks like https://tailwindcss.com/

**Comment by HenriBeck** [comment]
So you need something like this?

```js
const styles = {
  container: {
    '& > .item': {
      composes: 'container-item',
    },
  },
};
```

**Comment by HenriBeck** [comment]
I don't think this is possible without some overhead and bugs. We would need to run query selectors for applying the classnames.
